.github/workflows/azure-static-web-apps-thankful-mushroom-08ecc5d1e.yml

```
@@ -1,47 +1,52 @@
+  build_and_deploy_job:
+    if: github.event_name == 'push' || (github.event_name == 'pull_request' && github.event.action != 'closed')
+    runs-on: ubuntu-latest
+    name: Build and Deploy Job
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          submodules: true
+          lfs: false
+
+      - name: Fix git config permissions
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2046:warning:1:15: Quote this to prevent word splitting

(shellcheck)